.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```

Comment:
I'm not sure including the `vcs.xml` from your IDE is required here, we're interested in the code more than the setup

Reply:
I must have mistakenly added these when PyCharm made a suggestion, JetBrain configuration files will be removed.